cypress/component/Menu.spec.tsx

```diff
@@ -170,40 +167,44 @@ describe('Menu', () => {
         });
 
         it.skip('should focus on the third item', () => {
```

Comment:
do we still want to skip this test?

Reply:
This does not work currently in local either. Not sure if this is a bug or if this is something that wasn't a feature.